index.html

```diff
@@ -17,15 +17,23 @@
 <body>
   <header class="toolbar">
     <a class="logo">Omar Gutierrez</a>
-    <nav class="hamburger">
-      <img src="./images/Hamburger.svg" alt="menu">
+    <nav class="nav_toolbar">
+      <img src="./images/Hamburger.svg" alt="menu" class="hamburger" >
+      <ul class="nav-links">
+        <li class="nav-li"><a href="#">About</a></li>
+        <li class="nav-li"><a href="#">Work</a></li>
+        <li class="nav-li"><a href="#">Contact</a></li>
+        <li class="nav-li"><a href="#"><img src="./images/mail_icon.svg" alt="mail" class="mail_icon"></a></li>
+      </ul>
     </nav>
   </header>
   <section class="headline">
-    <h1 class="greeting">Hey There. I'm Omar.</h1>
-    <h3 class="presentation">I'm a Software Developer</h3>
-    <p class="description">I will help you build a product, feature or website. Look through some of my work and
-      experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
+    <div class="content">
+      <h1 class="greeting">Hey There. I'm Omar.</h1>
+      <h3 class="presentation">I'm a Software Developer</h3>
+      <p class="description">I will help you build a product, feature or website. Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
+    </div>
```

Comment:
- [ ] Hello you really did a great job by implemented some required changes from your previouse reviewer but there are still some changes requested from the previouse reviewer that you skip.
- [ ] According to your previous reviewer YOu currently have an horizontal scrollbar showing for the width of the viewport and this could be as a result of the use of excess margin in your css styling please kindly look into your design and fix it accordingly 👍

![Screenshot 2023-06-18 074240](https://github.com/ommarz/Personal_Portfolio/assets/92458236/a02a7e27-fac6-4a22-913e-b3edb0e9a103)